src/NUnitFramework/tests/TestUtilities/TestAssert.cs

```diff
@@ -81,7 +81,7 @@ public static void IsNotRunnable(Type type)
 
         public static void IsNotRunnable(Type type, string name)
         {
-            IsNotRunnable(TestBuilder.MakeTestCase(type, name));
+            IsNotRunnable(TestBuilder.MakeTestFromMethod(type, name));
```

Comment:
The test utilities are a bit adhoc. Could do with some reorganizing. Basically, they started out before we had parameterized test cases and were only ever changed to the extent needed.